libmavconn/include/mavconn/interface.h

```diff
@@ -44,6 +44,9 @@ using lock_guard = std::lock_guard<std::recursive_mutex>;
 //! Same as @p mavlink::common::MAV_COMPONENT::COMP_ID_UDP_BRIDGE
 static constexpr auto MAV_COMP_ID_UDP_BRIDGE = 240;
 
+//! Companion computer process component default ID
+static constexpr auto MAV_COMP_PROCESS_ID_DEFAULT = 1;
+
```

Comment:
Not needed. User is able to change link's component_id via params or url.

Reply:
@vooon I think you didn't understand the concept: We don't want to specify a unique component ID. We want to specify multiple component ID's, which can be parsed by this method, not by a parameter.